src/main/java/org/opentripplanner/routing/api/request/request/filter/AllowAllTransitFilter.java

```diff
@@ -0,0 +1,29 @@
+package org.opentripplanner.routing.api.request.request.filter;
```

Comment:
I think you can addd my hole comment in a `package.md` in this package. 

https://github.com/opentripplanner/OpenTripPlanner/issues/4440#issuecomment-1328993167

Reply:
I think we should do something like this:
![New Filter API](https://user-images.githubusercontent.com/5525340/211033597-2b766325-943f-4bd9-8c93-17db4b3c8756.png)